xtask/lintdoc/src/main.rs

```diff
@@ -306,6 +308,11 @@ fn generate_rule(
         writeln!(content)?;
     }
 
+    if let Some(source) = source {
+        writeln!(content, "Source: {source}")?;
```

Comment:
`SourceKind::Inspired` should write `Inspired from:`. `SourceKind::SameLogic` should write `Source:`

Reply:
How could I do this? The match statement? If so, what would it be like?

Reply:
Yeah, you use the match, and then you use the `write!()` macro instead. The macro will write in the buffer, but it won't add a new line at the end of the string.